packages/stark-core/src/http/services/http.service.ts

```diff
@@ -343,6 +350,17 @@ export class StarkHttpServiceImpl<P extends StarkResource> implements StarkHttpS
 			httpResponse$ = this.addRetryLogic(httpResponse$, retryCount);
 		}
 
+		let dummyError: Error = new Error(starkHttpServiceName + ": Error getting a CollectionResponse");
+		if (!dummyError.stack) {
+			// IE 11 won't generate a stack unless the error is thrown
+			// https://docs.microsoft.com/en-us/scripting/javascript/reference/stack-property-error-javascript#remarks
+			try {
+				throw new Error(starkHttpServiceName + ": Error getting a CollectionResponse");
```

Comment:
I think you can simply throw the same error you defined just above ("dummyError") instead of creating a new error again.